test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractUpdateSuite.java

```diff
@@ -93,7 +97,39 @@ public List<HapiSpec> getSpecsInSuite() {
                 eip1014AddressAlwaysHasPriority(),
                 immutableContractKeyFormIsStandard(),
                 updateAutoRenewAccountWorks(),
-                updateStakingFieldsWorks());
+                updateStakingFieldsWorks(),
+                consTimeManagementWorksWithRevertedInternalCreations());
+    }
+
+    private HapiSpec consTimeManagementWorksWithRevertedInternalCreations() {
```

Comment:
nit: We could probably move this to ContractCallSuite

Reply:
Done.